pkg/apis/serving/v1alpha2/inferenceservice_validation.go

```diff
@@ -32,6 +33,7 @@ const (
 	TrafficBoundsExceededError          = "TrafficPercent must be between [0, 100]."
 	TrafficProvidedWithoutCanaryError   = "Canary must be specified when CanaryTrafficPercent > 0."
 	UnsupportedStorageURIFormatError    = "storageUri, must be one of: [%s] or match https://{}.blob.core.windows.net/{}/{} or be an absolute or relative local path. StorageUri [%s] is not supported."
+    InvalidKSVCNameError                = "The InferenceService %s is invalid: the name must consist of lower case alphanumeric characters or '-', and must start with alphabetical character. (e.g. 'example-com')"
```

Comment:
Nit: your indentation is off. I recommend installing auto gofmt

Reply:
Thank you for the tip, I installed vim-go, cool!!